src/main/java/com/minecolonies/commands/MinecoloniesCommand.java

```diff
@@ -17,6 +17,8 @@
     private final ImmutableMap<String, ISubCommand> subCommands =
       new ImmutableMap.Builder<String, ISubCommand>()
         .put("colonies", new ColoniesCommand(DESC))
```

Comment:
Actually we cant, as this is a special case of only one level of parent. If we have nested SplitCommands they need all their parents